src/main/java/loanbook/logic/commands/RemindCommand.java

```diff
@@ -0,0 +1,106 @@
+package loanbook.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static loanbook.commons.util.CollectionUtil.requireAllNonNull;
+import static loanbook.logic.parser.CliSyntax.PREFIX_BIKE;
+import static loanbook.logic.parser.CliSyntax.PREFIX_NAME;
+import static loanbook.logic.parser.CliSyntax.PREFIX_PASSWORD;
+
+import java.util.List;
+
+import javax.mail.AuthenticationFailedException;
+
+import loanbook.commons.core.Messages;
+import loanbook.logic.CommandHistory;
+import loanbook.logic.SendReminder;
+import loanbook.logic.commands.exceptions.CommandException;
+import loanbook.model.Model;
+import loanbook.model.bike.Bike;
+import loanbook.model.loan.Loan;
+import loanbook.model.loan.LoanStatus;
+import loanbook.model.loan.Name;
+
+/**
+ * Send a reminder email to the customer.
+ */
+public class RemindCommand extends Command {
+
+    public static final String COMMAND_WORD = "remind";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Send a reminder email to the customer.\n"
+            + "Parameter: "
+            + PREFIX_PASSWORD + "EMAILPASSWORD "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_BIKE + "BIKE\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_PASSWORD + "hispassword "
+            + PREFIX_NAME + "John Doe "
```

Comment:
There is no need to remind for both the name and bike. One should imply the other.